### PR TITLE
Use the new cga2d_ni in more places

### PR DESCRIPTION
**Pull request by eric-wieser**
Follow-up to 2e368ab70c4bb44da6556d67e96b4d6c8408949d which defined `cga2d_ni = this.Coeff(4,1).Sub(this.Coeff(3,1))`

### CODE REVIEW

File: ganja.js [hunk 1]
```diff
@@ -640,17 +640,17 @@
...
+                var loc=o.Div(cga2d_ni.LDot(o)); lx=sc*(-loc.e1); ly=sc*(loc.e2); var r=-o.Mul(o.Conjugate).s/(Element.Pow(cga2d_ni.LDot(o),2).s); r=r**0.5*sc; return `<CIRCLE onmousedown="this.parentElement.sel=${oidx}" cx="${lx}" cy="${ly}" r="${r}" stroke-width="${options.lineWidth*0.005||0.005}" fill="none" stroke="${color||'green'}"/>`;
               } else if (!b1 && b2 &&!b3) {
               // Point Pairs.
-                lr=0; var ei=Element.Coeff(4,1,3,-1),eo=Element.Coeff(4,.5,3,.5), nix=o.Wedge(ei), sqr=o.LDot(o).s/nix.LDot(nix).s, r=Math.sqrt(Math.abs(sqr)), attitude=((ei.Wedge(eo)).LDot(nix)).Normalized.Mul(Element.Scalar(r)), pos=o.Div(nix); pos=pos.Div( pos.LDot(Element.Sub(ei)));
+                lr=0; var ei=cga2d_ni,eo=Element.Coeff(4,.5,3,.5), nix=o.Wedge(ei), sqr=o.LDot(o).s/nix.LDot(nix).s, r=Math.sqrt(Math.abs(sqr)), attitude=((ei.Wedge(eo)).LDot(nix)).Normalized.Mul(Element.Scalar(r)), pos=o.Div(nix); pos=pos.Div( pos.LDot(Element.Sub(ei)));
```
Comment (eric-wieser) [inline ganja.js]: `ni` or `ei`?

File: ganja.js [hunk 2]
```diff
@@ -640,17 +640,17 @@
               if (b1 && !b2 && !b3) {
                 o = o.Scale(-1/o.Dot(cga2d_ni).s);
                 lx=sc*(o.e1); ly=sc*(-o.e2); lr=0; return res2=`<CIRCLE onmousedown="this.parentElement.sel=${oidx}" cx="${lx}" cy="${ly}" r="${options.pointRadius*0.03||0.03}" fill="${color||'green'}"/>`;
-              } else if (!b1 && !b2 && b3) { var isLine=Element.Coeff(4,1,3,-1).LDot(o).Length==0;
+              } else if (!b1 && !b2 && b3) { var isLine=cga2d_ni.LDot(o).Length==0;
               // Lines.
                 if (isLine) {
-                  var loc=((Element.Coeff(4,-.5).Add(Element.Coeff(3,-.5))).LDot(o)).Div(o), att=(Element.Coeff(4,1,3,-1)).LDot(o);
+                  var loc=((Element.Coeff(4,-.5).Add(Element.Coeff(3,-.5))).LDot(o)).Div(o), att=cga2d_ni.LDot(o);
```
Comment (eric-wieser) [inline ganja.js]: Which of `(Element.Coeff(4,-.5).Add(Element.Coeff(3,-.5)` (here) and `Element.Coeff(4,.5,3,.5)` (line 653) is `e0`?

Comment (enkimute) [reply ganja.js]:
```
ni = 1e4 - 1e3
no = .5e4 + .5e3
```
I've apparently used -no too (probably to save some extra negations). 

All of this is up for change tho, as we are currently undertaking an effort to standardize the several Witt basis conventions.

I also wanted to provide support for custom ni and no, will need some time to make the needed changes. And I should change all of them to the extended Coeff syntax and save some operations.

File: ganja.js (hunk 1 above)
Comment (enkimute) [reply ganja.js]: ei is old.. new convention is ni,no, to emphasize that they are null vectors. (and I should change these)

**Pull request merged**